.github/actions/gcp-auth/action.yaml

```diff
@@ -0,0 +1,66 @@
+name: Authenticate against GAR
+description: |
+  Authenticates against GCP using OIDC. This action defines a default Service-Account as
+  identity.
+inputs:
+  gcp_service_account:
+    description: The Service-Account to impersonate
+    required: false
+    default: ci-opensource-operator@gardener-project.iam.gserviceaccount.com
+  workload_identity_provider:
+    description: |
+      GCP Workload Identity Provider. The default value exclusively allows GitHub-Actions from
+      github.com/gardener (no forks).
+    required: false
+    default: projects/694386720375/locations/global/workloadIdentityPools/github-actions/providers/github-actions-provider
+  gcp_project:
+    description: The GCP Project to authenticate against
+    required: false
+    default: gardener-project
+  gar_hostname:
+    description: |
+      The hostname of the GAR (for generating docker-cfg). For convenience, it is also
+      okay to pass a full image-reference (in which case hostname is parsed from parsed value).
+      Note that no special-handling for docker-hub (e.g. `alpine` ->
+      registry-1.docker.io/library/alpine:latest) is done.
+    required: false
+    default: europe-docker.pkg.dev
+outputs:
+  access_token:
+    description: The obtained access-token
+    value: ${{ steps.authenticate.outputs.access_token }}
```

Comment:
I'm wondering, how is this working? I'm not seeing any step with id `authenticate`, and the "Export Access-Token" step does not define any id 🤔

Reply:
@8R0WNI3 : good catch - fixed